src/main/packages/flowchart/flowchart-decision/flowchart-decision.tsx

```diff
@@ -2,12 +2,15 @@ import { FlowchartElementType } from '..';
 import { ILayer } from '../../../services/layouter/layer';
 import { ILayoutable } from '../../../services/layouter/layoutable';
 import { UMLElement } from '../../../services/uml-element/uml-element';
+import { calculateNameBounds } from '../../../utils/name-bounds';
 import { UMLElementType } from '../../uml-element-type';
 
 export class FlowchartDecision extends UMLElement {
   type: UMLElementType = FlowchartElementType.FlowchartDecision;
 
   render(canvas: ILayer): ILayoutable[] {
+    const namedBounds = calculateNameBounds(this, canvas);
+    this.bounds = namedBounds;
```

Comment:
I would personally skip the creation of the constant and just assign to this.bounds directly, as we are not using namedBounds anywhere else